rest/changes_api_test.go

```diff
@@ -198,6 +198,117 @@ func TestDocDeletionFromChannel(t *testing.T) {
 
 }
 
+//Simulate two SG instances to test rev_cache side effects
+func TestDocDeletionFromChannelTwoSG(t *testing.T) {
+	// See https://github.com/couchbase/couchbase-lite-ios/issues/59
+	// base.LogKeys["Changes"] = true
+	// base.LogKeys["Cache"] = true
+
+	rt1 := RestTester{SyncFn: `function(doc) {channel(doc.channel)}`}
+	defer rt1.Close()
+
+	a := rt1.ServerContext().Database("db").Authenticator()
+
+	// Create user in rt1:
+	alice, _ := a.NewUser("alice", "letmein", channels.SetOf("zero"))
+	a.Save(alice)
+
+	rt2 := RestTester{RestTesterBucket: rt1.RestTesterBucket, SyncFn: `function(doc) {channel(doc.channel)}`}
```

Comment:
The TTL used for old revisions is now configurable via old_rev_expiry_seconds property in DbConfig.

We can use this in the unit tests to expire the tombstone rev without waiting for 5 minutes